js/options.js

```diff
@@ -7,99 +7,133 @@
 'use strict';
 
 function loadVals() {
-
-    const popupcolor = localStorage['popupcolor'];
-    for (let i = 0; i < document.optform.popupcolor.length; i++) {
-        if (document.optform.popupcolor[i].value === popupcolor) {
-            document.optform.popupcolor[i].selected = true;
+    switch (localStorage['popupcolor']) {
+        case 'blue':
+            document.querySelector('#popupColorBlue').checked = true;
+            break;
+        case 'lightblue':
+            document.querySelector('#popupColorLightBlue').checked = true;
```

Comment:
There is a bug here regarding the camel casing. The ID of the input element is "popupColorLightblue". If popupcolor === 'lightblue' then the loadVals() function will throw an exception here and nothing will be selected on the options page.

I think, ideally we should use camel case, such as lightBlue. However, the current code base unfortunately stores 'lightblue' in lowercase. So I guess the easiest - and most consistent - way to fix this bug is to just change the query selector to #popupColorLightblue here.

Reply:
Good catch. Sorry about that.